src/apicast-cli/blank-app/blank-app.rockspec

```diff
@@ -0,0 +1,10 @@
+package = "blank-app"
```

Comment:
Shouldn't this be the project name ?

I guess we can leave it for now and use liquid templating when it is integrated properly.